hal/lib/athena/HALAthena.cpp

```diff
@@ -5,8 +5,6 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "HAL/HAL.h"
```

Comment:
Since we don't have a HAL.cpp, we need to include it first somewhere.  This seems as good a place as any...